docs/developer-docs/smart-contracts/deploy/overview.mdx

```diff
@@ -17,26 +17,30 @@ After you have built a <GlossaryTooltip>canister</GlossaryTooltip>, you can
 deploy it to:
 
 - <strong>The local canister execution environment</strong> on your machine where
-  no tokens/cycles are required
-- <strong>The playground on mainnet</strong>, a testnet-like sandbox
-  environment. No tokens or cycles are required to deploy and run your canister
-  smart contract but it will be removed after 20 minutes.
+  no tokens or cycles are required
+- <strong>The playground on the mainnet</strong>, a testnet-like sandbox
+  environment. No tokens or cycles are required to deploy and run your canister
+  smart contract, but it will be removed after 20 minutes.
 - <strong>Mainnet</strong> for production use. This deployment method will require
-  cycles.
+  cycles.
 
 This page will guide you through all 3 deployment methods.
 
 ## Testnets
 
 Most blockchain networks have separate testnets that allow smart contract
 developers to test their projects in a production-like environment at
-significantly lower cost than on mainnet. Because the execution of canisters is
-fairly cheap on ICP, there is no testnet for ICP. Developers are encouraged to
-test their canister smart contracts locally or directly on the mainnet. In order to
-make test deployments to mainnet faster and even cheaper, you can deploy your
-canister to the so-called playground. You do not need cycles or tokens to deploy and
-operate your canister in the playground, but it will be automatically removed
-after 20 minutes.
+significantly lower cost than on the mainnet. Because the execution of canisters is
+fairly cheap on ICP, and ICP smart contracts can be upgraded once deployed, there is no testnet for ICP.
+Developers are encouraged to test their canister smart contracts locally or directly on the mainnet.
+
+However, the [playground](/docs/current/developer-docs/getting-started/deploy/testnet) sandbox environment can be used as a testnet-like deployment option. It does not require cycles or tokens to deploy and operate a canister, but it has limitations, and canisters will be automatically removed after 20 minutes.
+
+For most developers, the playground option can be used for most workflows, as will be demonstrated below. For more advanced developers and use cases, there are two possible options for a testnet-like environment:
```

Comment:
the language here might be a tad repetitive since the word `most` is being used twice

Reply:
Updated to "For the majority of developers..."